docs/stability.rst

```diff
@@ -298,7 +298,7 @@ The current planned and existing sub-packages are:
         </tr>
         <tr>
             <td>
-                astropy.vo.samp
+                astropy.samp
```

Comment:
Maybe it is clearer to state in the description that it was renamed from ``astropy.vo.samp`` in v2.0? Other entries have info like that (when added, etc).